README.md

```diff
@@ -44,6 +44,8 @@ npm install -g create-react-app
 
 **We strongly recommend to use Node >= 6 and npm >= 3 for faster installation speed and better disk usage.** You can use [nvm](https://github.com/creationix/nvm#usage) to easily switch Node versions between different projects.
 
+**Note:** `nvm` does not support Windows (nvm issue [#284](https://github.com/creationix/nvm/issues/284)). See potential [alternatives](https://github.com/creationix/nvm#installation).
```

Comment:
What if I remove the note on support for Windows users and rather update the link for nvm to direct users to the installation section of the docs which explains its compatibility. The reason I added the note is that the link to nvm takes the user to the usage section which doesn't quickly let the user see that they may not be able to use this for node management. 

**We strongly recommend to use Node >= 6 and npm >= 3 for faster installation speed and better disk usage.** You can use [nvm](https://github.com/creationix/nvm#installation) to easily switch Node versions between different projects.